lib/context.js

```diff
@@ -61,8 +61,15 @@ api.process = ({activeCtx, localCtx, options}) => {
 
     // reset to initial context
     if(ctx === null) {
-      rval = activeCtx = api.getInitialContext(options);
-      continue;
+      if(termContext) {
+        rval = activeCtx = api.getInitialContext(options);
+        continue;
+      } else {
+        throw new JsonLdError(
```

Comment:
Hmm, I don't think this is quite right. We want to allow context nullification anywhere -- the exception is when sealed terms are defined. Sealed terms are not cleared unless via a term definition for a term that is used to extend the JSON tree.

Reply:
That seems surprising to me in that it's introducing a lot more edge-case'y behavior...

Reply:
It shouldn't be -- we're targeting how we clear `@sealed` terms only. `@context` nullification for regular terms is a JSON-LD 1.0 feature that we can't change (for backwards compatibility reasons).

Reply:
Based on our IRC conversation I think I've corrected this to your satisfaction.